### ISSUE TITLE
[Feature Request] Autosave On Change 

### ISSUE THREAD

**Issue by zhaolinlau** [atom/autosave#104]
I am here to request for autosave on change feature because autosave on change can make coding life much easier and convenient <3

**Comment by kasac** [comment]
Dear @zhaolinlau,


I believe, there is a package for that: 

https://atom.io/packages/autosave-onchange


I hope, I got your request right. :)

**Comment by zhaolinlau** [comment]
> Dear @zhaolinlau,
> 
> 
> I believe, there is a package for that: 
> 
> https://atom.io/packages/autosave-onchange
> 
> 
> I hope, I got your request right. :)

Nope, I'm requesting for adding this feature as built-in feature.
